components/dashboard/src/App.test.ts

```diff
@@ -0,0 +1,18 @@
+/**
+ * Copyright (c) 2021 Gitpod GmbH. All rights reserved.
+ * Licensed under the GNU Affero General Public License (AGPL).
+ * See License-AGPL.txt in the project root for license information.
+ */
+
+import { getURLHash } from './App'
+
+test('urlHash', () => {
+    global.window = Object.create(window);
+    Object.defineProperty(window, 'location', {
+        value: {
+            hash: '#example.org'
+        }
+    });
+
+    expect(getURLHash()).toBe('example.org');
```

Comment:
**Optional**: Maybe we can make the example URL look more like a contextURL:

```suggestion
            hash: '#https://example.org/user/repo'
        }
    });

    expect(getURLHash()).toBe('https://example.org/user/repo');
```

But this is entirely optional of course. 👍